packages/vike-react/src/renderer/onRenderClient.tsx

```diff
@@ -5,12 +5,15 @@ import ReactDOM from 'react-dom/client'
 import { getHeadSetting } from './getHeadSetting.js'
 import type { OnRenderClientSync } from 'vike/types'
 import { getPageElement } from './getPageElement.js'
+import './styles.css'
 
 let root: ReactDOM.Root
 const onRenderClient: OnRenderClientSync = (pageContext): ReturnType<OnRenderClientSync> => {
   // Use case:
   // - Store hydration https://github.com/vikejs/vike-react/issues/110
-  pageContext.config.onBeforeRenderClient?.(pageContext)
+  for (const onBeforeRenderClient of pageContext.config.onBeforeRenderClient || []) {
```

Comment:
:+1: Neat, although let's do it in another PR. Technically, it's a (small) breaking change (but I think it's fine we release it in a patch).